.github/workflows/step_lifecycle.yaml

```
@@ -0,0 +1,104 @@
+          fi
+
+          echo "pending_updates=${pending_updates}" >> "$GITHUB_OUTPUT"
+          
+      - name: Merge PR
+        if: ${{ steps.find_pr.outputs.pending_updates != '' }}
+        run: |
+          echo "Merging PR ${{ steps.find_pr.outputs.pending_updates }}"
+          gh pr merge -s ${{ steps.find_pr.outputs.pending_updates }}
+  
+  rebuild:
+    if: needs.merge.outputs.pr == "" && needs.prev_step.outputs.step == "main"
```

actionlint (1.7.7)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)